packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts

```diff
@@ -125,4 +129,10 @@ export interface IBasePickerSuggestionsProps {
    * Indicates whether to show a button with each suggestion to remove that suggestion.
    */
   showRemoveButtons?: boolean;
+}
+
+export enum validationState {
```

Comment:
Nit: I believe all of our enums start with capitol letters so this should be ValidationState.